### PR TITLE
add inventory parser

### PR DESCRIPTION
**Pull request by murtaza-webonise**

### CODE REVIEW

File: inventory/src/main/java/com/sungardas/wwss/impl/InventoryReaderImpl.java [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+import com.sungardas.wwss.models.InventoryItem;
+import com.sungardas.wwss.models.InventoryMetadata;
+import com.sungardas.wwss.models.Resource;
+
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class InventoryReaderImpl implements InventoryReader {
```
Comment (ameyawebonise) [inline inventory/src/main/java/com/sungardas/wwss/impl/InventoryReaderImpl.java:17]: you want to rename it to FilePathInventoryReader?this way we can switch the impl later on

File: inventory/pom.xml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.sungardas.wwss</groupId>
+    <artifactId>inventory</artifactId>
```
Comment (ameyawebonise) [inline inventory/pom.xml:6]: can we name it to inventory-loader?

File: inventory/src/test/java/com/sungardas/wwss/InventoryTest.java [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.List;
+
+public class InventoryTest {
+  
+  @Test
+  public void inventoryListShouldNotBeEmpty() throws IOException {
+    String fileName = "/home/webonise/Desktop/workspace/constraint-optimiser/inventory/src/main/java/com/sungardas/wwss/inventory.json";
```
Comment (ameyawebonise) [inline inventory/src/test/java/com/sungardas/wwss/InventoryTest.java:16]: this is an absolute file name.Not everyone will have the same resource at that path.can we load it from the classpath?

Comment (murtaza-webonise) [reply inventory/src/test/java/com/sungardas/wwss/InventoryTest.java:16]: My bad missed it

File: inventory/src/main/java/com/sungardas/wwss/impl/InventoryReaderImpl.java (hunk 1 above)
Comment (murtaza-webonise) [reply inventory/src/main/java/com/sungardas/wwss/impl/InventoryReaderImpl.java:17]: yes

File: inventory/pom.xml (hunk 2 above)
Comment (murtaza-webonise) [reply inventory/pom.xml:6]: yes

File: inventory-loader/pom.xml [hunk 4]
```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.sungardas.wwss</groupId>
+    <artifactId>inventory</artifactId>
```
Comment (ameyawebonise) [inline inventory-loader/pom.xml:6]: should it not be `inventory-loader`

**Review by ameyawebonise — APPROVED** [review]

**Pull request merged**